virtinst/cloner.py

```diff
@@ -455,6 +458,23 @@ def set_nvram_path(self, val):
     # Functional methods #
     ######################
 
+    def _prepare_serial_files(self):
+        for serial in chain(self._new_guest.devices.console,
+                            self._new_guest.devices.serial):
+            if serial.type != DeviceSerial.TYPE_FILE:
+                continue
+
+            def cb_exists(path):
+                # Ignore the check for now
+                return False
+
+            serial.source.path = _generate_clone_path(self.src_name,
+                                                      self.new_guest.name,
+                                                      serial.source.path,
+                                                      cb_exists=cb_exists)
+            Path(serial.source.path).touch()
```

Comment:
This bit is still making local host changes, even though this could be a remote operation.

Is the `touch` required? In my testing, qemu and/or libvirt will create those log files on VM startup if they don't exist.

Reply:
Yes, sorry, that is a leftover for name collision resolution. I've removed it.